AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst

```diff
@@ -0,0 +1,103 @@
+.. _AuxTel-Non-Standard-Operations-AuxTel-Non-standard-Operations: 
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+#########################
+AuxTel Emergency Shutdown
+#########################
+
+.. _Title-of-Procedure-Overview:
+Overview
+========
+.. This section should provide a brief, top-level description of the procedure's purpose and utilization. Consider including the expected user and when the procedure will be performed.
+Unfavorable circumstances that indicate the need to close the dome if it is open are:
+   - `Weather constraints`_ 
+   - Sunrise approaching in the next two hours
+   - Power outage affecting the dome shutter mechanism
+ 
+
+.. _Title-of-Procedure-Precondition:
+Precondition
+============
+.. This section should provide simple overview of preconditions before executing the procedure; for example, state of equipment, telescope or seeing conditions or notifications prior to execution.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. If there is a different procedure that is critical before execution, carefully consider if it should be linked within this section or as part of the Procedure section below (or both).
+The dome shutter is open and an unforeseeable condition arises, forcing an emergency dome closure.
+
+.. _Title-of-Procedure-Post-Condition:
+Post-Condition
+==============
+.. This section should provide a simple overview of conditions or results after executing the procedure; for example, state of equipment or resulting data products.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Please provide screenshots of the software status or relevant display windows to confirm.
+.. Do not include actions in this section. Any action by the user should be included in the end of the Procedure section below. For example: Do not include "Verify the telescope azimuth is 0 degrees with the appropriate command." Instead, include this statement as the final step of the procedure, and include "Telescope is at 0 degrees." in the Post-condition section.
+- The emergency situation has been clearly notified to the relevant personnel and management.
+- Steps to eliminate or minimize the risk to the instruments and hardware are being implemented.
+- Eventually, the dome shutter is closed and all equipment is safe.
+
+.. _Title-of-Procedure-Procedure-Steps:
+Procedure Steps
+===============
+
+.. todo::
+   Please use the standard safety measures when visiting the dome enclosure. **Safety of personnel always goes first.**
```

Comment:
This comment is not visible on the page https://obs-ops.lsst.io/v/SITCOM-1361/AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.html - am I looking at the right branch?

Reply:
Added to the existing warning.